torchvision/ops/boxes.py

```diff
@@ -167,25 +164,35 @@ def box_convert(boxes: Tensor, in_fmt: str, out_fmt: str) -> Tensor:
             boxes_xyxy = _box_xywh_to_xyxy(boxes)
             if out_fmt == "cxcywh":
                 boxes_converted = _box_xyxy_to_cxcywh(boxes_xyxy)
+            else:
+                raise ValueError("Unsupported Bounding Box Conversions for given in_fmt and out_fmt")
```

Comment:
You don't actually need those to make torchscript happy. The problem is that `boxes_converted` is not defined in the other branch.
You can define it outside of the `if` so that torchscript knows its type.